index.js

```diff
@@ -84,9 +84,12 @@ export class UpdateAPK {
     // You might check {totalSpace, freeSpace} = await RNFS.getFSInfo() to make sure there is room
     const downloadDestPath = `${RNFS.CachesDirectoryPath}/NewApp.apk`;
 
+    let headers = this.options.apkHeaders ? this.options.apkHeaders : {};
```

Comment:
this would change to match any change above

Reply:
I changed this and using Object.assign in downloadFile the user can complements the param or replace them.